pkg/config/config.go

```diff
@@ -138,7 +138,8 @@ func init() {
 	BindEnvAndSetDefault("forwarder_num_workers", 1)
 	// Dogstatsd
 	Datadog.SetDefault("use_dogstatsd", true)
-	Datadog.SetDefault("dogstatsd_port", 8125)          // Notice: 0 means UDP port closed
+	Datadog.SetDefault("dogstatsd_port", 8125) // Notice: 0 means UDP port closed
+	Datadog.SetDefault("bind_host", "localhost")
```

Comment:
nit: what so you think about moving this up in the `"Agent"` section since it's not meant to be dogstatsd-specific?